### PR TITLE
Added fit.py

### PR DESCRIPTION
**Pull request by AReyH**
* Added the fit function.
* Created the docstring for the function

### CODE REVIEW

File: src/autopredictor/fit.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+def fit(X_train,X_test,y_train,y_test):
+
```
Comment (zgarciaj) [inline src/autopredictor/fit.py]: I might need to double check but I believe there is no space needed in between def and the triple quote.
```suggestion
```

File: src/autopredictor/fit.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    """
+    Train and evaluate multiple regression models on the given training and test data.
```
Comment (zgarciaj) [inline src/autopredictor/fit.py:3]: Should we specify the models that we will be fitting?

**Review by zgarciaj — CHANGES_REQUESTED** [review]
Great work Arturo! I made a minor suggestion on spacing and potential addition to the initial description. Otherwise, I believe the function is ready for merger.

File: src/autopredictor/fit.py (hunk 2 above)
Comment (AReyH) [reply src/autopredictor/fit.py:3]: I don't think so because we are not yet sure which ones we are using. I feel like we can change them for Milestone 2.

Comment (zgarciaj) [reply src/autopredictor/fit.py:3]: Sounds good!

**Review by zgarciaj — APPROVED** [review]

**Pull request merged**